### PR TITLE
[one-cmds] Fix OptionBuilder

### PR DESCRIPTION
**Pull request by jinevening**
This enables to use correct optimization options.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/onelib/OptionBuilder.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def _build_optimize(self, commands):
                 options.extend(['--' + k])
                 continue
             for opt in CONSTANT.OPTIMIZATION_OPTS:
-                if k == opt and v == "True":
+                if k == opt[0] and v == "True":
```
Comment (seanshpark) [inline compiler/one-cmds/onelib/OptionBuilder.py:63]: (this is not related with this PR but... @mhs4670go , iv `v` is not related with this for loop, can we hoist `v == 'True'` outside of this loop? in another PR)

**Comment by seanshpark** [comment]
`opt` and `opt[0]` looks very different, but does python work as same?
how did it work?

**Comment by jinevening** [comment]
Previously, optimization options were not applied at all. So this is a bug fix.

**Comment by seanshpark** [comment]
> Previously, optimization options were not applied at all. So this is a bug fix.

Wow... this is a serious thing...
was the usage like
```
onecc --optiona --optionb ...
```
but `optiona` nor `optionb` was applied for optimize ?

File: compiler/one-cmds/onelib/OptionBuilder.py (hunk 1 above)
Comment (mhs4670go) [reply compiler/one-cmds/onelib/OptionBuilder.py:63]: Sorry. I don't get it. Could you explain what you said with simple example?

**Review by mhs4670go — APPROVED** [review]
LGTM

**Comment by mhs4670go** [comment]
This is a bug of workflows features. I think related tests should be added.

File: compiler/one-cmds/onelib/OptionBuilder.py (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/onelib/OptionBuilder.py:63]: as `v` is not related any variable inside the loop,
```py
if v == "True":
  for opt in CONSTANT.OPTIMIZATION_OPTS:
    if k == opt[0]:
      options.extend(['--' + k])
      break
```

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/one-cmds/onelib/OptionBuilder.py (hunk 1 above)
Comment (mhs4670go) [reply compiler/one-cmds/onelib/OptionBuilder.py:63]: We can:) I will post a PR soon.

**Pull request merged**